package.json

```diff
@@ -132,6 +132,7 @@
     "regenerator-runtime": "^0.13.3",
     "rrweb": "^1.1.3",
     "rrweb-player": "^0.7.13",
+    "screenfull": "^6.0.1",
```

Comment:
New dep: MIT license. 

It's about 150 lines if we wanted to copy+paste it instead

https://www.npmjs.com/package/screenfull
https://github.com/sindresorhus/screenfull